ContentFiles/Forms/FormsUIContent/HomeView.xaml.pp

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<views:MvxContentPage
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:views="clr-namespace:MvvmCross.Forms.Views;assembly=MvvmCross.Forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:$rootnamespace$.Views"
+    x:Class="$rootnamespace$.Views.HomeView">
```

Comment:
If we were to use the strong typed version, this file needs to be edited to include `x:TypeArguments="viewModels:HomeViewModel"` and `xmlns:viewModels="clr-namespace:$rootnamespace$.ViewModels;assembly=$rootnamespace$"`